stumpy/core.py

```diff
@@ -568,6 +569,10 @@ def check_window_size(m, max_size=None):
     max_size : int, default None
         The maximum window size allowed
 
+    n : int, default None
+        The length of the time series in the case of a self-join.
+        `n` should be set to `None` in the case of an `AB-join`.
```

Comment:
"``n`` should not be supplied (or set to ``None``) in the case of an ``AB-join``"

Note that for docstrings, all variables MUST be enclosed in TWO backticks

Reply:
> "`n` should not be supplied (or set to `None`) in the case of an `AB-join`"

👍 

> Note that for API docstrings (that are shown to users in our readthedocs), all variables MUST be enclosed in TWO backticks instead of SINGLE backticks.

Didn't know that! Thanks for the info. Will fix it.

Reply:
@seanlaw 

> > Note that for API docstrings (that are shown to users in our readthedocs), all variables MUST be enclosed in TWO backticks instead of SINGLE backticks.
> 
> Didn't know that! Thanks for the info. Will fix it.

I just noticed that the function `core.check_window_size` is not shown in [STUMPY's readthedocs](https://stumpy.readthedocs.io/en/latest/index.html). Having said that, I can follow the two-backtick rule regardless if the function is shown to the user in the readthedocs or not.

Reply:
@seanlaw 
Okay. I pushed the changed and enclosed variables with TWO backticks. I've assumed you are okay with this:

> I can follow the two-backtick rule regardless if the function is shown to the user in the readthedocs or not.

If not, please let me know and I can revert the change.